packages/gatsby/src/redux/actions/public.ts

```diff
@@ -1112,21 +1143,25 @@ actions.replaceWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
  *   }
  * })
  */
-actions.setBabelOptions = (options: Object, plugin?: ?Plugin = null) => {
+actions.setBabelOptions = (
+  options: Record<string, any>,
+  plugin: IGatsbyPlugin | null = null
+): ISetBabelOptionsAction => {
   // Validate
-  let name = `The plugin "${plugin.name}"`
-  if (plugin.name === `default-site-plugin`) {
-    name = `Your site's "gatsby-node.js"`
-  }
-  if (!_.isObject(options)) {
+  const pluginPlaceholder = `Your site's "gatsby-node.js"`
+  const hasNoPluginName = !plugin || plugin.name === `default-site-plugin`
+  const name = `The plugin "${
+    hasNoPluginName ? pluginPlaceholder : plugin!.name
+  }"`
```

Comment:
this results in a different name if the plugin is the default-site-plugin. So need to change that.

Reply:
This is identical to the logic before (= if name is `default-site-plugin` it is changed to `Your site's "gatsby-node.js"`. Should I fix that?